pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1964,7 +1969,7 @@ func DomainCreate(status types.DomainStatus) (int, string, error) {
 
 // Launch app/container thru rkt
 // returns domainID, podUUID and error
-func rktRun(domainName string, ContainerImageID string, xenCfgFilename string) (int, string, error) {
+func rktRun(domainName string, imageUUID string, ContainerImageID string, xenCfgFilename string) (int, string, error) {
```

Comment:
Why not just pass the ACI file name here instead of imageUUID string, ContainerImageID ?? Wouldn't that be better?